src/sections/partners/Partners.content.tsx

```diff
@@ -1,45 +1,92 @@
-import su from "../../assets/images/logos/su.jpg";
-import PartnerItem from "../../types/TPartner";
+import PartnerItem from '../../types/TPartner';
+import buurger from '../../assets/images/sponsors/buurger.jpg';
+import hame from '../../assets/images/sponsors/hame.jpg';
+import klenotyAurum from '../../assets/images/sponsors/klenotyAurum.jpg';
+import ponozkovice from '../../assets/images/sponsors/ponozkovice.svg';
+import LGZ from '../../assets/images/sponsors/LGZ.png';
+import lucy from '../../assets/images/sponsors/lucy.png';
+import patifu from '../../assets/images/sponsors/patifu.jpg';
+import topNatur from '../../assets/images/sponsors/topNatur.svg';
+import vitar from '../../assets/images/sponsors/vitar.svg';
+import vitco from '../../assets/images/sponsors/vitco.png';
+import infinitMaximus from '../../assets/images/sponsors/infinitMaximus.svg';
 
 export const partners: PartnerItem[] = [
   {
-    name: "Studentská Unie UTB",
-    logoSrc: su,
-    link: "https://sunovy.utb.cz/",
+    name: 'flyUnited',
+    logoSrc:
+      'https://www.flyunited.cz/wp-content/themes/flyunited/Assets/img/logotype.svg',
```

Comment:
for now OK, but normally it's better to approach to save the logo locally and serve it like that instead of a link (if there is a small change in the URL in the future, the image will be broken)